FPMUL/FPMUL_OutputMonitor.py

```diff
@@ -13,11 +13,13 @@ def __init__(self, dut, txn_valid, clock, callback=None, event=None):
 
     @coroutine
     def _monitor_recv(self):
+        self.dut._log.info("OutputMonitor received something")
         clk_edge = RisingEdge(self.clock)
 
         while True:
             yield clk_edge
-            if self.txn_valid:
+            if self.txn_valid is True: # ntwong0 - looks like you need to explicitly specify that you are checking this parameter against True, because Python

```

Comment:
`self.txn_valid` might be an object of a different type. 

`self.txn_valid.value` may also work